### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by herulume**
~~I still can't configure 'now', I don't have the proper permissions, @nelsonmestevao address this asap.~~

# Problems: 
- My css-fu isn't enough to put the footer at the bottom of the page.
- When loading the facebook icon gets really huge (still looking into this).
- Some inline parameters are passed that way instead of using pure Styled Components since they are either not valid css, or I'm dumb (most likely this is the reason).

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -1,21 +1,78 @@
...
+import {
+  faFacebook,
+  faTwitter,
+  faInstagram,
+  faGithub,
+} from '@fortawesome/free-brands-svg-icons';
+
+
+const PaddingBox = styled(Box)({
+  padding: 1,
```
Comment (nelsonmestevao) [inline components/Footer.js]: You should probably increase this.

**Comment by herulume** [comment]
The icons getting huge seems to only happen when the website is not cached/on the first load.

**Comment by a-sac** [comment]
@herulume muda a cor dos icons quando o cursor passa por cima pls 🙏

**Comment by herulume** [comment]
@a-sac  Can't do it. Feel free to push to this branch

**Comment by alves-luis** [comment]
It now displays a different color when hovered. Not sure about the color to choose, feel free to leave suggestions.

File: components/Footer.js [hunk 2]
```diff
@@ -1,21 +1,85 @@
...
+    </PaddingBox>
+  </Grid>
+);
+
+const WithLove = () => (
+  <Grid item xs={12}>
+    <CenteredBox mb={1}>
+      <Typography>made with </Typography>
+      <Favorite color="error" style={{ fontSize: 20 }}>Love</Favorite>
+      <Typography> by CeSIUM</Typography>
```
Comment (PedroFerr101) [inline components/Footer.js]: I would suggest making the word CeSIUM clickable and redirect it to CeSIUM's webpage.

**Pull request merged**